src/components/form/head.vue

```diff
@@ -264,7 +264,6 @@ $tab-li-margin-top: 5px;
       "tabTitle": "Tyto funkce budou k dispozici, jakmile zveřejníte svůj koncept formuláře"
     },
     "draftNav": {
-      "title": "Koncept",
```

Comment:
This translation still exists, it was just moved to the locale files in src/locales/ (as part of #1001).